eva-accession-pipeline/src/test/resources/accession-pipeline-test.properties

```diff
@@ -13,8 +13,8 @@ parameters.vcf=src/test/resources/input-files/vcf/small_genotyped.vcf.gz
 parameters.vcfAggregation=NONE
 
 parameters.fasta=src/test/resources/input-files/fasta/Homo_sapiens.GRCh37.75.chr20.head_1200.fa
-#parameters.outputVcf=/tmp/accession-output.vcf
-parameters.outputVcf=C:/Users/sundarvenkata/AppData/Local/Temp/accession-output.vcf
+parameters.outputVcf=/tmp/accession-output.vcf
+#parameters.outputVcf=C:/Users/sundarvenkata/AppData/Local/Temp/accession-output.vcf
```

Comment:
Please don't leave commented lines in the code, specially if they refer to particular users' machines.